# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/util-linux-fseeko.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./text-utils/pg.c.orig	2009-02-09 11:08:13.000000000 +0100
+++ ./text-utils/pg.c	2009-03-29 18:11:08.000000000 +0200
@@ -166,7 +166,7 @@
 See pg(1) for more information.\n\
 -------------------------------------------------------\n";
 
-#ifndef HAVE_FSEEKO
+#if !defined(HAVE_FSEEKO) && !defined(__UCLIBC__)
   static int fseeko(FILE *f, off_t off, int whence) {
 	return fseek(f, (long) off, whence);
   }
